src/main/java/com/google/netpcapanalysis/dao/GeolocationDaoImpl.java

```diff
@@ -0,0 +1,40 @@
+package com.google.netpcapanalysis.dao;
+
+import com.google.netpcapanalysis.interfaces.dao.GeolocationDao;
+import com.maxmind.geoip2.DatabaseReader;
+import com.maxmind.geoip2.model.CityResponse;
+import com.maxmind.geoip2.record.Country;
+import java.io.File;
+import java.io.IOException;
+import java.net.InetAddress;
+import java.net.URL;
+
+public class GeolocationDaoImpl implements GeolocationDao {
+
+  private static final String GEO_DB_LOCATION = "GeoLite2-City.mmdb";
```

Comment:
Is this not under `data/` according to your README?

Reply:
It's mapped through Java resources.